src/Components/Profile.css

```diff
@@ -6,6 +6,8 @@
 }
 
 .p-avatar-image > img {
+  width: 4rem;
```

Comment:
can't this be done with prime-react classes?

Reply:
Avatar is the parent component of the image,
But the image is not exposed , I don't know how to apply custom CSS to a child component from its parent component